docs/index.bs

```diff
@@ -1563,6 +1563,9 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
       dictionary RouterCondition {
         URLPatternCompatible urlPattern;
+        USVString requestMethod;
```

Comment:
requestMethod in the fetch standard is defined as `ByteString` [1]. I wonder if it's fine to define this as USVString or not. 
[1] https://fetch.spec.whatwg.org/#request-class

Reply:
Matching the types from Fetch is best, so I agree that making it `ByteString` would be better.

This has slight observable consequences as it means that `requestMethod: "あ"` will throw immediately, instead of simply failing to match. A test for that might be good.

Reply:
good catch!
I overlooked that.
I will also fix the Chromium code X(.

Reply:
FYI: Chromium fix is https://chromium-review.googlesource.com/c/chromium/src/+/5223724